pallets/stream-payment/Cargo.toml

```diff
@@ -46,6 +49,7 @@ std = [
 	"frame-system/std",
 	"log/std",
 	"pallet-balances/std",
+	"pallet-migrations?/std",
```

Comment:
The way we usually do it is the following:

- in the pallet, we just write the funciton that migrates
- in the runtime/common we write a custom struct that implements the trait from pallet-migrations. This allows us to not depend on pallet-migrations in the pallets